.github/workflows/fullnode-execute-devnet-stable.yaml

```diff
@@ -0,0 +1,57 @@
+# This workflow runs a public fullnode using the `devnet` branch,
+# connects the public fullnode to `devnet` and synchronizes the
+# node using execution syncing to verify that nothing has been broken.
+
+name: "fullnode-execute-devnet-stable"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 9/12 * * *"
+  pull_request:
+    paths:
+      - ".github/workflows/fullnode-execute-devnet-stable.yaml"
+
+jobs:
+  fullnode-execute-devnet-stable:
+    timeout-minutes: 300 # Run for at most 5 hours
+    runs-on: high-perf-docker-with-local-ssd
+    steps:
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3
+
+      - uses: ./.github/actions/fullnode-sync
+        with:
+          TIMEOUT_MINUTES: 300 # Run for at most 5 hours
+          BRANCH: devnet
+          NETWORK: devnet
```

Comment:
Ah, devnet waypoint will not be updated in the network github repo tho, since it got wiped every week, we still use the url to get waypoint and genesis

Reply:
Shouldn't be hard to upload waypoint and genesis to repo each release though? fwiw I always remember to do it :p https://github.com/aptos-labs/aptos-networks/commit/d079eb9d6d4ba8d2c8cb049aa73ed21022a3de6a

Reply:
Yeah, I think this is something we should fix because our dev docs instruct users to do this: https://aptos.dev/nodes/full-node/fullnode-source-code-or-docker#method-1-building-and-running-from-source. So, in a way, having this test is good, because if we fail to do it, the test will fail and I'll poke someone 😄